docs/internal/Assignment-Process.md

```diff
@@ -27,14 +27,12 @@ Review assignment can have the following statuses
 
 To also determine if this assignment can be Self-assigned, 3 other fields come on the scene:
 
-- `is_self_assignable` -> Can be self assigned (if not locked - how we prevent other self-assigners after one is already assigned)
-- `is_locked` -> Can be used to either prevent self-assignment or to generate a Review with status = LOCKED & assignment status is ASSIGNED\*
+- `is_self_assignable` -> Can be self assigned (if there is `available_sections` to be selected - how we prevent other self-assigning for sections already assigned)
+- `is_locked` -> Used to prevent a Review from being submitted - in case another reviewer in same stage already send a response to Applicant (decision to "Send back to applicant")
 
 Another possibility for assignment is using the flag:
 
-- `is_final_decision` -> Used to assign reviewers with specific permissions that gets Assignment.status = ASSIGNED automatically.
-
-\* This is specifically used for assignment of partial reviews. For example, 2 reviewers have been assigned to different Sections of the same application. After one submits back to the Applicant the other one needs to be blocked -- this would be flagged in the Review.status = LOCKED. But if there is only Assignment and no review started, we set the Assignment is_locked = true to generate a Review.status LOCKED when started.
+- `is_final_decision` -> Used to assign reviewers to do a single-decision review. In this case the review will be always `is_self_assignable` and `is_last_level
```

Comment:
Updated reference to `is_final_decision` since now we do assign the review first and disable other `review_assignment` for that stage, using the same logic similar to any other review (that is self-assignable)